Dockerfile

```diff
@@ -45,6 +43,10 @@ RUN cp -r /usr/local/boring/go/ /usr/local/go/
 COPY --from=builder /usr/local/go/bin/go-assert-boring.sh /usr/local/go/bin/go-assert-static.sh /usr/local/go/bin/go-build-static.sh /usr/local/go/bin/
 COPY --from=builder $GOPATH/src/github.com/projectcalico/calico $GOPATH/src/github.com/projectcalico/calico
 
+FROM builder AS calico-node-builder-s390x
```

Comment:
Could you add a comment on top of this line saying `# We need a different image for s390x because goboring installation is not supported`

Reply:
Done in https://github.com/rancher/image-build-calico/pull/21/commits/70351c99610e9242378603ff061b937b5b3e3791